src/ui/Ingredient.tsx

```diff
@@ -0,0 +1,54 @@
+import Select from './Select';
+import { Field } from 'formik';
+import Input from './Input';
+import styled from '@emotion/styled';
+import { FC } from 'react';
+
+const StyledDiv = styled.div`
+  color: var(--text);
+  height: 51px;
+  display: flex;
+  justify-content: flex-left;
+  gap: 10px;
+  align-items: center;
+  margin-bottom: 10px;
+`;
+
+const StyledField = styled(Field)`
+  border-radius: 0px;
+  border-bottom: 1px var(--text) solid;
+  height: 40px;
+  width: 10%;
+  padding-left: 10px;
+  background: var(--inputbackground);
+  margin-bottom: 10px;
+  color: var(--text);
+`;
+
+const Ingredient: FC = ({ arrayHelpers, index }) => {
```

Comment:
make sure you're typing your props, your IDE should flag this as an error, but lmk if it's not for some reason.

```
interface IngredientProps {
  arrayHelpers: string; // i don't know what type this is supposed to be
  index: number;
}
```

Reply:
`const Ingredient: FC<IngredientProps> = ...`